src/module/helpers/processResponseData.js

```diff
@@ -18,21 +20,22 @@ import { isMissingModule, registerMissingModule } from './missingModule'
  * @param {String} currentModule
  * @param {Object} data
  */
-export function processResponseData (vuexInstance, vuexFns, api, currentModule, data, currentMethod = '') {
-  for (let destinationModule in data) {
-    if (!data.hasOwnProperty(destinationModule)) {
+export function processResponseData (vuexInstance, vuexFns, api, currentModule, data, currentMethod = '', module) {
```

Comment:
Module should have a default value since it's after a parameter with default value.